README.md

```
@@ -1,239 +1,27 @@
-MIT
-
-## Contributing
-
-Feel free to use this template as a starting point for your own TurboWarp/Scratch extensions!
-
-For questions or improvements, open an issue or pull request.
+1. Open [TurboWarp](https://turbowarp.org/editor)
+2. Click the 'Add extension' button (the puzzle piece and the plus icon)
+3. Search for 'Custom Extension' and click on its result
+4. Click 'Files' and 'No files selected'
+5. When your file explorer of choice opens, find the downloaded file and double-click it
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)